packages/app/cypress/e2e/top-nav.cy.ts

```diff
@@ -1,5 +1,6 @@
 import type { SinonStub } from 'sinon'
 import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
+import { CYPRESS_REMOTE_MANIFEST_URL, NPM_CYPRESS_REGISTRY_URL } from '@packages/types/src'
```

Comment:
is the `/src` necessary on this import path?

Reply:
No, I don't think so. Removed